### DISCUSSION TITLE
Can I add an image source?

### DISCUSSION THREAD

**Discussion by akspic** [xdan/jodit#967] [Q&A] [answered]
Hey there!
 In articles, I need to add an image source, but I did not find such an option during image upload. How can I add a source or a short description?

<img width="830" alt="image" src="https://user-images.githubusercontent.com/22916656/232420925-34edbfb0-cca6-49da-bede-8c31cecd697e.png">

**Comment by ashishsigdel** [comment]
You can make base64 image uploader by this line of code: 
const editor = Jodit.make("#editor", {
  "uploader": {
    "insertImageAsBase64URI": true
  }
});

and also you can add image from external url default image selectbutton

**Comment by xdan** [comment, marked as answer]
There's no dedicated source/caption field — the image dialog only has alt and title, and neither shows on the page (alt is for accessibility, title is just the hover tooltip). For a visible caption or source line you'd add it as text under the image, or wrap the image in a `<figure>` with a `<figcaption>` — Jodit keeps that markup, it just doesn't insert it for you.

If you'd want a built-in caption field in the image dialog, worth opening a feature request so it can be considered.
